src/Analyzers/CSharp/Analyzers/UseDefaultLiteral/CSharpUseDefaultLiteralDiagnosticAnalyzer.cs

```diff
@@ -46,19 +48,13 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
 
             // Create a normal diagnostic that covers the entire default expression.
             context.ReportDiagnostic(
-                DiagnosticHelper.Create(
+                DiagnosticHelper.CreateWithLocationTags(
                     Descriptor,
                     defaultExpression.GetLocation(),
                     preference.Notification.Severity,
-                    additionalLocations: null,
-                    properties: null));
-
-            // Also fade out the part of the default expression from the open paren through 
-            // the close paren.
-            context.ReportDiagnostic(
-                Diagnostic.Create(
-                    UnnecessaryWithoutSuggestionDescriptor,
-                    syntaxTree.GetLocation(fadeSpan)));
+                    additionalLocations: new[] { defaultExpression.SyntaxTree.GetLocation(fadeSpan) },
+                    tagIndices: ImmutableDictionary<string, IEnumerable<int>>.Empty
+                        .Add(nameof(WellKnownDiagnosticTags.Unnecessary), new int[] { 0 })));
```

Comment:
never changes.  perhaps make static for clarity?

Reply:
➡️ Will be a natural simplification as part of https://github.com/dotnet/roslyn/pull/46666#discussion_r470892118